week-2/interpret/time-format.js

```diff
@@ -26,18 +26,23 @@ console.log(formatTimeDisplay(143));
 // Questions
 
 // a) When formatTimeDisplay is called how many times will pad be called?
+// Ans. 3
 
 // Call formatTimeDisplay with an input of 143, now answer the following:
 
 // b) What value is assigned to the parameter num when pad is called for the first time?
-
+//Ans. 0
 // c) What is the return value of pad when it is called for the first time?
-
+// Ans. "00"
 // d) What is the value assigned to the parameter num when pad
 // is called for the last time in this program?  Explain your answer
+//Ans. 
```

Comment:
Looks like part of this answer is missing

Reply:
I have added the answer